embedded-graphics/src/fonts/font12x16.rs

```diff
@@ -27,98 +27,8 @@ impl FontBuilderConf for Font12x16Conf {
 ///
 /// # Examples
 ///
-/// ## Write some text to the screen at the default `(0, 0)` position
+/// See the [module docs](./index.html) for examples.
```

Comment:
The official rust docs always use `[module-level documentation]` for this link, which I also used in the image module docs.

Reply:
Thanks for this - I've updated the PR to be consistent with rust docs and the `image` module.